deploy/unifi.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env sh
+
+#Here is a script to deploy cert to unifi controller. Based on:
+# https://community.ubnt.com/t5/UniFi-Wireless/Your-own-SSL-key-and-cert/m-p/484943#M39260
+
+#returns 0 means success, otherwise error.
+
+########  Public functions #####################
+
+#domain keyfile certfile cafile fullchain
+unifi_deploy() {
+  _cdomain="$1"
+  _ckey="$2"
+  _ccert="$3"
+  _cca="$4"
+  _cfullchain="$5"
+
+  _debug _cdomain "$_cdomain"
+  _debug _ckey "$_ckey"
+  _debug _ccert "$_ccert"
+  _debug _cca "$_cca"
+  _debug _cfullchain "$_cfullchain"
+
+  openssl pkcs12 -export -in "$_ccert" -inkey "$_ckey" \
```

Comment:
do not use `openssl` command explicitly.  we have a function `toPkcs()`,  you can refactor that function and reuse it.